scripts/smoke-test.sh

```
@@ -0,0 +1,330 @@
+#
+# Requirements: mcporter, nc, curl, jq (or python3)
+
+set -euo pipefail
+
+# ─── Config ──────────────────────────────────────────────────────────────────
+MCP_URL="${SYSLOG_MCP_URL:-http://localhost:3100/mcp}"
+HEALTH_URL="${MCP_URL%/mcp}/health"
+SYSLOG_HOST="${SYSLOG_HOST:-127.0.0.1}"
+SYSLOG_PORT="${SYSLOG_PORT:-1514}"
+SKIP_SEED=0
+MCPORTER_ARGS=""
```

Shellcheck (0.11.0)

[warning] MCPORTER_ARGS appears unused. Verify use (or export if used externally).

(SC2034)